src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -147,5 +147,47 @@ public IDataView FilterByKeyColumnFraction(IDataView input, string columnName, d
                 throw Environment.ExceptSchemaMismatch(nameof(columnName), "filter", columnName, "KeyType", type.ToString());
             return new RangeFilter(Environment, input, columnName, lowerBound, upperBound, false);
         }
+
+        /// <summary>
+        /// Shuffle the rows of <paramref name="input"/>.
+        /// </summary>
```

Comment:
From this comment I don't understand if the output `IDataView` will be of size `poolRows` or if it will be of the same size as the input `IDataView`

Reply:
How about now? I rewrote the remarks to be a bit more explicit on what was happening.

---
In reply to: [254190313](https://github.com/dotnet/machinelearning/pull/2427#discussion_r254190313) [](ancestors = 254190313)

Reply:
It's very clear now thanks for explaining further!

---
In reply to: [254337002](https://github.com/dotnet/machinelearning/pull/2427#discussion_r254337002) [](ancestors = 254337002,254190313)